schemas/user/auth-user-summary.yml

```diff
@@ -22,20 +22,22 @@ properties:
 
   first_name:
     type: string
+    readOnly: true
     description: |
       The user's first name.
 
   last_name:
     type: string
+    readOnly: true
     description: |
       The user's last name.
 
-required:
-  - first_name
-  - last_name
+  permissions:
+    type: array
+    readOnly: true
+    items:
+      $ref: '#/definitions/permission/permission'
+    description: |
+      The permissions granted to the user.
```

Comment:
Hmm, I think changing this to be a `/permissions/` endpoint like seed-auth-api would be better. If we end up needing to retrieve data for the authenticated user we can add a `/user` endpoint for that. Thanks :)